lib/ood_core/job/adapters/slurm.rb

```diff
@@ -193,7 +193,8 @@ def accounts
                   name: acct,
                   qos: qos.to_s.chomp.split(','),
                   cluster: cluster,
-                  queue: queue.to_s.empty? ? nil : queue
+                  # ignore queue because it doesn't imply access, only resource limits
+                  # queue: queue.to_s.empty? ? nil : queue
```

Comment:
There's no need for the comment here if we're going to remove it.

Reply:
I'd say remove the comment because it points to the field `queue` that no longer exists in `AccountInfo`. In a year, it'd be a rabbit-hole to nowhere because queues no longer exist.

As to the command and column - sure we can remove them now or later.